src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -0,0 +1,48 @@
+@import 'src/styles/common';
+
+.row {
+  display: grid;
+  grid-template-columns: [label] 120px [value] auto;
+  grid-column-gap: 10px;
+  margin-bottom:3px;
+
+  .label {
+    grid-column: label;
+    color: $dark-grey;
+    text-align: right;
+  }
+
+  .value {
+    grid-column: value;
+  }
+}
+
+
+.spaceAbove{
+  margin-top: 30px;
+}
+
+
+.featureDetails {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+  flex-wrap: nowrap;
+  overflow: hidden;
+
+  & > div:not(:first-child) {
+    margin-left: 30px;
+  }
+}
+
+.featureSymbol {
+  font-weight: $bold;
+}
+.stableId{
+  font-weight: $bold;
+}
+.featureSymbol + .stableId{
+  margin-left: 12px;
+  color: $dark-grey;
```

Comment:
I believe it shouldn't be gray since it's light? Compare how it looks:

<img src="https://user-images.githubusercontent.com/6834224/107517428-a52e8f80-6ba5-11eb-96f3-8cba0ba70b69.png" width="300px" />

with the way similar elements look in the transcript drawer:

<img src="https://user-images.githubusercontent.com/6834224/107517455-aeb7f780-6ba5-11eb-8785-1b656ce0dc02.png" width="400px" />

Reply:
Yes, I missed to update it here. That's done